src/columns/fast-formula-parser.ts

```diff
@@ -0,0 +1,85 @@
+import * as glide from "../glide";
+
+import FormulaParser, {FormulaHelpers, Types, FormulaError, MAX_ROW, MAX_COLUMN} from 'fast-formula-parser';
+
+var data: any[][] = [];
```

Comment:
You can't keep global data.  `run` is async and can run concurrently multiple times, which means that the separate invocations would overwrite each other's `data`.